osu.Game.Rulesets.Taiko/Skinning/Default/CirclePiece.cs

```diff
@@ -50,25 +50,16 @@ public Color4 AccentColour
 
                 background.Colour = AccentColour;
 
-                resetEdgeEffects();
+                background.EdgeEffect = new EdgeEffectParameters
+                {
+                    Type = EdgeEffectType.Glow,
+                    Colour = AccentColour.Opacity(kiaiMode ? 0.5f : 1f),
+                    Radius = kiaiMode ? 32 : 8
+                };
             }
         }
 
-        private bool kiaiMode;
-
-        /// <summary>
-        /// Whether Kiai mode effects are enabled for this circle piece.
-        /// </summary>
-        public bool KiaiMode
-        {
-            get => kiaiMode;
-            set
-            {
-                kiaiMode = value;
-
-                resetEdgeEffects();
-            }
-        }
+        private static bool kiaiMode;
```

Comment:
I'm 99.999% sure you didn't intend to make this static.

Reply:
Fixed!